src/modules/Security.rb

```diff
@@ -545,24 +555,24 @@ def write_shadow_config
 
     # Write settings related to PAM behavior
     def write_pam_settings
-      # use cracklib?
-      if @Settings["PASSWD_USE_CRACKLIB"] == "yes"
-        Pam.Add("cracklib")
+      # use pwquality?
+      if @Settings["PASSWD_USE_PWQUALITY"] == "yes"
+        Pam.Add(pwquality_module)
         pth = @Settings["CRACKLIB_DICT_PATH"]
         if pth && pth != "/usr/lib/cracklib_dict"
-          Pam.Add("--cracklib-dictpath=#{pth}")
+          Pam.Add(pwquality_module + "-dictpath=#{pth}")
         end
       else
-        Pam.Remove("cracklib")
+        Pam.Remove(pwquality_module)
```

Comment:
here I am not sure if it is correct. What happens when user use old cracklib and now new pwquality is available? I think it is safer to remove both here, not?

Reply:
Well, switching from pam_cracklib to pam_pwquality will be the task of pam. YaST is not involved here.

Anyway, already now adding cracklib adds pwquality (if pam-deprecated is installed). I was not able to add cracklib anymore so removing it seems useless to me.